bedrock/firefox/templates/firefox/includes/download-button.html

```diff
@@ -4,7 +4,7 @@
 
 {% macro alt_buttons(builds) %}
   <div class="download download-dumb">
-    <h4>{{ _('Download Firefox') }} — {{ locale_name|safe }}</h4>
+    <p role="heading" class="download-heading"><b>{{ _('Download Firefox') }} — {{ locale_name|safe }}</b></p>
```

Comment:
Should we go with `strong` for emphasis? Maybe the `heading` role alone implies enough importance, in which case we shouldn't need the `b` either.